library/init/meta/interactive.lean

```diff
@@ -6,8 +6,11 @@ Authors: Leonardo de Moura
 prelude
 import init.meta.tactic init.meta.rewrite_tactic init.meta.simp_tactic
 import init.meta.smt.congruence_closure init.category.combinators
+import init.meta.lean_parser
 
 namespace interactive
+@[reducible] meta def parse {α : Type} (p : lean_parser α) : Type := α
```

Comment:
> Ah, I see. Could we insert a type annotation instead?

Where would we insert it? Adding a type annotation around `(pexpr.subst (pexpr.subst ...))` would not help.
BTW, I want to emphasize I'm fine with this issue. In my point of view, an incorrect parser action is a programming error.